dev-docs/source/RunningSanitizers.rst

```diff
@@ -0,0 +1,156 @@
+.. _RunningSanitizers:
+
+##################
+Running Sanitizers
+##################
+
+.. contents::
+    :local:
+
+Overview
+=========
+
+Sanitizers allow developers to find any issues with their code at runtime
+related to memory, threading or undefined behaviour.
+
+Pre-requisites
+==============
+
+The following tooling is required:
+
+- CMake 13.0 onwards (instructions for Ubuntu `here <https://apt.kitware.com/>`__)
```

Comment:
Typo here I think? `3.13.0`?

Reply:
:+1: - we can drop this all-together now that the project minimum is 3.13 anyway